Reduce fixture valueset duplication

c-monkey opened the pull request:
Fixtures updated to only contain the required value sets as specified in value_set_oid_version_objects. Also updated the fixtures slightly as required for Bonnie-1248

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] This PR is into the correct branch.
- [X] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1248
- [X] JIRA ticket links to this PR
- [X] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [X] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [X] Tests are included and test edge cases
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [X] Automated regression test(s) pass

**Reviewer into feature branch**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

mayerm94 commented on lib/tasks/cql_testing.rake:
```diff
@@ -28,10 +28,18 @@ namespace :bonnie do
+
+      measure.value_set_oid_version_objects.each do |vs_v|
+        db_value_sets = HealthDataStandards::SVS::ValueSet.where(user_id: measure.user_id, oid: vs_v['oid'], version: vs_v['version'])
+        puts 'FAILED to find value set' unless db_value_sets.exists?
```
Include info about what value set/user could not be found. Rake task will error out if db_value_sets cannot be found so we will probably want to know which value set on which user it is erroring out on

mayerm94 commented on lib/tasks/cql_testing.rake:
```diff
@@ -96,11 +105,13 @@ namespace :bonnie do
-      value_sets.each do |vs|
+      measure.value_set_oid_version_objects.each do |vs_v|
+        db_value_sets = HealthDataStandards::SVS::ValueSet.where(user_id: orig_user_id, oid: vs_v['oid'], version: vs_v['version'])
+        puts 'FAILED to find value set' unless db_value_sets.exists?
```
Same as first comment

c-monkey commented:
@mayerm94 I have addressed your feedback. Here is sample output:
```
ctohline$ bundle exec rake bonnie:fixtures:generate_frontend_cql_fixtures[cms,CQL/CMS32,bonnie-fixtures@mitre.org,CMS32v7]
exported measure to spec/javascripts/fixtures/json/measure_data/CQL/CMS32/CMS32v7.json

FAILURE!!!!!

FAILED to find value set for:
	user: bonnie-fixtures@mitre.org
	oid: 2.16.840.1.113762.1.4.1
	version: Draft-3FD13096-2C8F-40B5-9297-B714E8DE9133xxx

FAILURE!!!!!
```
